.github/workflows/test_runner.yaml

```diff
@@ -35,19 +33,27 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.8"
-      - name: Install
+      - name: Install dpes
         run: |
-          set -xe
-          python -m pip install --upgrade pip
-          python -m venv venv
-          venv/bin/pip install --progress-bar=off --requirement requirements.txt
+            touch requirements.*.txt  # ensure we don't rebuild
```

Comment:
Might be good to expand this comment to explain why touching this file will ensure no rebuild.  I've followed the current dev stack and it still took me a second to work out why this would be the case!

Reply:
done